app/components/carousel/carousel.tsx

```diff
@@ -0,0 +1,98 @@
+'use client'
+
+import React from 'react'
+import Slider from 'react-slick'
+import { Box, IconButton, useBreakpointValue } from '@chakra-ui/react'
+import { BiLeftArrowAlt, BiRightArrowAlt } from 'react-icons/bi'
+
+const settings = {
+  dots: true,
+  arrows: false,
+  fade: true,
+  infinite: true,
+  autoplay: true,
+  speed: 500,
+  autoplaySpeed: 5000,
+  slidesToShow: 1,
+  slidesToScroll: 1
+}
+
+export const Carousel = () => {
+  const [slider, setSlider] = React.useState<Slider | null>(null)
+
+  const top = useBreakpointValue({ base: '90%', md: '50%' })
+  const side = useBreakpointValue({ base: '30%', md: '10px' })
+
+  // These are the images used in the slide
+  const cards = [
+    'https://images.unsplash.com/photo-1612852098516-55d01c75769a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1yZWxhdGVkfDR8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=900&q=60',
+    'https://images.unsplash.com/photo-1627875764093-315831ac12f7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1yZWxhdGVkfDJ8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=900&q=60',
+    'https://images.unsplash.com/photo-1571432248690-7fd6980a1ae2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1yZWxhdGVkfDl8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=900&q=60'
+  ]
+
+  return (
+    <Box
+      position={'relative'}
+      height={'400px'}
+      width={'full'}
+      overflow={'hidden'}
+    >
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css"
+      />
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css"
+      />
+      {/* Left Icon */}
+      <IconButton
+        aria-label="left-arrow"
+        bgColor="gray.400"
+        colorScheme="white"
+        borderRadius="full"
+        position="absolute"
+        left={side}
+        top={top}
+        transform={'translate(0%, -50%)'}
+        zIndex={2}
+        onClick={() => slider?.slickPrev()}
+        _hover={{ opacity: 0.8 }}
+      >
+        <BiLeftArrowAlt />
+      </IconButton>
+      {/* Right Icon */}
+      <IconButton
+        aria-label="right-arrow"
+        bgColor="gray.400"
+        colorScheme="white"
+        borderRadius="full"
+        position="absolute"
+        right={side}
+        top={top}
+        transform={'translate(0%, -50%)'}
+        zIndex={2}
+        onClick={() => slider?.slickNext()}
+        _hover={{ opacity: 0.8 }}
+      >
+        <BiRightArrowAlt />
+      </IconButton>
```

Comment:
Can I ask you the reason why you used this icon? I think material/feather left and right icons better! (As we only used these icons)
If you can not find a good one, you can download from figma and make icon component. And this arrow style is better 🍎 

![Screenshot 2023-11-28 at 8 17 45 PM](https://github.com/tabi-memo/frontend/assets/66394413/2d865671-64e9-4d09-9ce9-ea27551c0695)

Reply:
Sorry all I did was copy and pasted the chakra carousel! I completely forgot to use the icon that we picked! 
It was the ones I made so I converted svg to jsx.